File: FISH/fish.py

n = int(input("Enter n: "))
for i in range(1,n+1):
    for j in range(1,n-i+1):
        print(" ",end="")

    for k in range(1,2*i):
        print("*",end="")

    for j in range(1, 2*(n-i)+1):
        print(" ",end="")

    for l in range(3,i+1):
        print("*",end="")
    print("")


for i in range(n-1, 0, -1):
    for j in range(1,n-i+1):
        print(" ",end="")

    for k in range(1,2*i):
        print("*",end="")

    for j in range(1, 2*(n-i)+1):
        print(" ",end="")

    for l in range(3,i+1):
        print("*",end="")
    print("")

File: FISH/fish.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cout << "Please enter n" << endl;
    cin >> n;
    //TOP HALF OF THE FISH
    for (int i = 0; i < n; i++){
        for (int j = 0; j < n - i; j++)
            cout<<" ";
        for (int j = 0; j < 2 * i - 1; j++)
            cout << "*";
        for (int j = 0; j < 2 * (n - i); j++)
            cout << " ";
        for (int j = 3; j <= i; j++)
            cout << "*";
        cout << endl;
    }
    //BOTTOM HALF OF THE FISH
    for (int i = n; i >= 1; i--){
        for (int j = 0; j < n - i; j++)
            cout << " ";
        for (int j = 0; j < 2 * i - 1; j++)
            cout << "*";
        for (int j = 0; j < 2 * (n - i); j++)
            cout << " ";
        for (int j = 3; j <= i; j++)
            cout << "*";
        cout << endl;
    }
    return 0;
}
